packages/frontend/src/components/Button/style.module.scss

```diff
@@ -1,9 +1,9 @@
 .button {
   align-items: center;
-  background-color: transparent;
+  background-color: var(--buttonSecondaryBackground);
   border-radius: 10px;
-  border: 0;
-  color: var(--buttonText);
+  color: var(--buttonSecondaryText);
+  border: 1px solid #ccc;
```

Comment:
This applies the border to primary buttons as well, which IMO looks a bit weird. But not a big deal I guess.

![image](https://github.com/user-attachments/assets/c88a136e-ff0f-4ec5-8b89-505740c0c2b0)

Reply:
Is fixed